src/Compilers/CSharp/Test/Symbol/Symbols/Source/DelegateTests.cs

```diff
@@ -712,9 +712,6 @@ class C
     D d = async delegate { };
 }";
             CreateCompilation(source).VerifyDiagnostics(
-                // (4,17): error CS1988: Async methods cannot have ref, in or out parameters
-                //     D d = async delegate { };
-                Diagnostic(ErrorCode.ERR_BadAsyncArgType, "delegate"),
```

Comment:
From offline discussion, @333fred pointed out we should still have an error in this scenario. I agree.